### ISSUE TITLE
CS-BOLD-032 Informational 7.14: Redemptions Can Lower the ICR

### ISSUE THREAD

**Issue by bingen** [liquity/bold#490]
The comment in the function `TroveManager.redeemCollateral` states that redemptions can only improve the CR of troves:
```
// Skip if ICR < 100%, to make sure that redemptions always improve the CR of hit Troves
if (getCurrentICR(singleRedemption.troveId, _price) < _100pct) {
    singleRedemption.troveId = nextUserToCheck;
    continue;
}
```

This assumption is based on the fact that during a redemption the equal value of collateral and debt is removed from a Trove. However, when the ICR is equal to 100% then the same amount of collateral and debt is burned and the ICR will stay constant. ICR = 100

Further, if the Trove is in a batch, then function `TroveManager._updateBatchShares` rounds down the number of batch shares that are burned from a user, so the amount of debt that is burned from the user can be slightly lower than the amount paid by the redeemer. This can reduce the ICR by a small amount.

In conclusion, it is not true that redemptions always improve the CR. This may lead to a redemption making a trove liquidatable that previously was not, or could make the branch enter shutdown.

**Comment by bingen** [comment]
> This may lead to a redemption making a trove liquidatable that previously was not, or could make the branch enter shutdown.

If the trove was >= 110% CR, then it would improve the CR, so it’s not true that the redemption can make the trove liquidatable (unless the rounding error of batch shares is huge).
Same applies for making the branch entering shutdown.
